django/applications/catmaid/control/common.py

```diff
@@ -35,13 +37,16 @@ class parsedict(dict):
     """
     pass
 
-def get_request_bool(request_dict, name, default=None):
+def get_request_bool(request_dict:Dict, name:Optional[str], default:Optional[bool]=None) -> Union[bool, None]:
     """Extract a boolean value for the passed in parameter name in the passed
     in dictionary. The boolean paramter is expected to be a string and True is
     returned if it matches the string "true" (case-insensitive), False otherwise.
+
+    If the name may not be present in the dictionary, caller must provide a default
+    value or the return value will be None rather than boolean.
```

Comment:
Thanks for being more explicit about the behavior here. We mimic basically what `dict.get()` does.